include/exadg/aero_acoustic/calculators/source_term_calculator.h

```diff
@@ -87,6 +95,25 @@ class SourceTermCalculator
 
     Number rho = static_cast<Number>(data.density);
 
+    // In case we blend in the source term we check, if the scaling is space dependent. Only in that
+    // case we have to evaluate the function in every equadrature point. Otherwise the scaling
+    // purely temporal and constant during this function.
```

Comment:
Just a minor comment: The comma in the first sentence should be after "source term". In the second setence, "is" seems to be missing.

Reply:
[033b96f](https://github.com/exadg/exadg/pull/644/commits/033b96fe0ecdd3fda9a0e7f11e8d4c3420cfd04b)